src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -1102,4 +1106,24 @@ a.tribe-rating-link {
 			text-decoration: underline;
 		}
 	}
+	.plugin-list {
+		display: inline-block;
+		font-weight: 600;
+		margin: 0;
+		padding: 0;
+
+		span.plugin-invalid {
+
+			&:after {
+				content: ', ';
+			}
+
+			&:last-of-type {
```

Comment:
I don't see any place `.last-of-type` is applied in the `is_api_*_warning()` methods - is it injected some other place not shown in this PR?